doc/source/cluster/vms/user-guides/faq.rst

```diff
@@ -0,0 +1,56 @@
+.. _cluster-FAQ:
+
+FAQ
+===
+
+What is the difference between ``--node-ip-address`` and ``--address``?
+-----------------------------------------------------------------------
+
+When starting a head node, ``ray start --node-ip-address xx.xx.xx.xx --port
```

Comment:
```suggestion
When starting a head node, ``ray start --head --node-ip-address xx.xx.xx.xx --port
```
Should we add `--head` to this command?

Also, a user might wonder when it is necessary to specify `--node-ip-address` and what happens when it isn't specified.  The naive question is why do we need to tell the head node its own ip address, and I guess the answer is that it's an *externally reachable* IP address, and Ray needs to know it to allow worker nodes to connect?

Reply:
I reworked this section and related to the other review comments.